app/services/puppet_fact_parser.rb

```diff
@@ -140,6 +140,14 @@ def cores
     facts.dig('processors', 'count') || facts['processorcount']
   end
 
+  def disks_total
+    disks = facts['disks']
+
+    return nil unless disks
+
+    disks.sum { |disk| disk&.fetch('size_bytes', 0).to_i }
```

Comment:
If we're going minimal, isn't this the same?
```suggestion
    facts['disks']&.sum { |disk| disk&.fetch('size_bytes', 0).to_i }
```